scripts/scp_api.py

```diff
@@ -241,9 +257,10 @@ def __init__(self):
 
         APIManager.__init__(self)
         print("INIT")
-        self.api = "https://portals.broadinstitute.org/single_cell/api/v1/"
+        self.api_base = None # set in APIManager.login()
         self.studies = None
         self.name_to_id = None
+        self.bucket_id = None
         self.species_genomes = {"cat":["felis_catus_9.0","felis_catus_8.0","felis_catus-6.2"]}
```

Comment:
Is there a reason these are hard coded?  In the future I would imagine that we could hit the taxons API endpoint to get the list of available species for an instance of the portal:  https://portals.broadinstitute.org/api/v1/taxons